.eslintrc.js

```diff
@@ -1,3 +1,3 @@
 /* eslint-env node */
 
-module.exports = require("@bleu-balancer-tools/eslint-config");
+module.exports = require("@bleu/eslint-config");
```

Comment:
Prefer `bleu-fi` so that we're able to eventually publish packages to github packages under [the same namespace.](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry)